guardian/interactive-atom-thrasher-template
# Replace The Fiver by Football Daily in football multi thrasher

## What does this change?

Update the multi football thrasher, replacing The Fiver by Football Daily

## How to test

Run `gulp` from your IDE
Select `default` from the list of Atoms then select one of the Thrasher containers or Snaps

To test the success/error message has inverted colours for the Football Daily thrasher, see the testing instructions in PR #39 

## Images

Before
![Screenshot 2022-11-08 at 11 26 24](https://user-images.githubusercontent.com/74301289/200553371-ccf7fe61-99c2-48a4-a505-30f5b66992e1.png)

After
![Screenshot 2022-11-08 at 11 22 33](https://user-images.githubusercontent.com/74301289/200553561-77bb1d89-01cb-4e36-8e55-7cd071cb7cac.png)
